.github/workflows/release.yml

```diff
@@ -20,24 +20,41 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      # Set up Java Environment
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      # Grant execute permission to the gradlew script
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      # Build the project with Gradle
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
+        run: |
+          ./gradlew build
+          
       # Build Ballerina Project
       - name: Ballerina Build
-        uses: ballerina-platform/ballerina-action/@master
+        uses: ballerina-platform/ballerina-action/@2201.0.3
         with:
           args:
             pack ./salesforce
         env:
-          CLIENT_ID: ${{ secrets.CLIENT_ID }}
-          CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
-          EP_URL: ${{ secrets.EP_URL }}
-          REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
-          REFRESH_URL: ${{ secrets.REFRESH_URL }}
+          JAVA_HOME: /usr/lib/jvm/default-jvm
+
       # Push to Ballerina Central
       - name: Ballerina Push
-        uses: ballerina-platform/ballerina-action/@master
+        uses: ballerina-platform/ballerina-action/@2201.0.3
```

Comment:
As discussed offline, lets use 2201.1.0 instead of 2201.0.3

Reply:
Ack fixed in fc08a84ae3b62f98c8863f7b4b430da95533b776